docker/runtime-fuzzer/scripts/fuzzer.sh

```diff
@@ -70,7 +70,7 @@ function start_container_post {
 		rustup component add llvm-tools-preview && \
 		rustup component add --toolchain nightly llvm-tools-preview && \
 		cargo fuzz coverage --release --sanitizer=none main /corpus/main -- \
-        -rss_limit_mb=8192 -max_len=35000000 -len_control=0 && \
+        -rss_limit_mb=8192 -max_len=20000000 -len_control=0 && \
```

Comment:
what about introducing .sh file with consts like this?

Reply:
partially done,
for docker related scripts - https://github.com/gear-tech/gear/issues/3397